internal/handlers/common/error.go

```diff
@@ -42,7 +42,8 @@ const (
 	ErrCommandNotFound   = ErrorCode(59)    // CommandNotFound
 	ErrInvalidNamespace  = ErrorCode(73)    // InvalidNamespace
 	ErrNotImplemented    = ErrorCode(238)   // NotImplemented
-	ErrSortBadValue      = ErrorCode(15974) // Location15974
+	ErrLocation15974     = ErrorCode(15974) // Location15974
```

Comment:
@ribaraka could you please give here some meaningful name? Thank you.

Reply:
sure! done.